packages/frontend-test-utils/src/app/createExtensionTester.tsx

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ComponentType, ReactNode, useContext, useState } from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { Link } from '@backstage/core-components';
```

Comment:
Thinking we could perhaps use react-router-dom here directly rather than our own link? I think it'd be better to avoid the analytics events anyway, and it would also be nice to avoid a dependency on `core-components` if we can.

Reply:
Good idea, done 🙌🏻